src/server.js

```diff
@@ -156,6 +164,15 @@ function indexGamSorted(req, res) {
   });
 }
 
+function endsWithExtensions(file, extensions) {
```

Comment:
It would be nice to have documentation comments for new functions and what they are supposed to accomplish.